### DISCUSSION TITLE
Mass addition to list

### DISCUSSION THREAD

**Discussion by Kovah** [Kovah/LinkAce#368] [Ideas & Feature Requests]
This topic continues the discussion of a post from the old community forum: "Mass addition to list" by Alexander_Stepanov
A backup of the full conversation can be viewed on the [Internet Archive](https://web.archive.org/web/20220130215746/https://community.linkace.org/t/mass-addition-to-list/158).

---

> It would be nice if there will be an option to mass add all links found by search to list

**Comment by amadeusp** [comment]
In this context it would also be great if one could make mass mutations (e.g. change tags or add to lists) in the "All Links" view, as well as in Tags and Lists view respectively.
